microk8s-resources/actions/common/utils.sh

```diff
@@ -117,3 +117,11 @@ produce_server_cert() {
     openssl req -new -key ${SNAP_DATA}/certs/server.key -out ${SNAP_DATA}/certs/server.csr -config ${SNAP_DATA}/certs/csr.conf
     openssl x509 -req -in ${SNAP_DATA}/certs/server.csr -CA ${SNAP_DATA}/certs/ca.crt -CAkey ${SNAP_DATA}/certs/ca.key -CAcreateserial -out ${SNAP_DATA}/certs/server.crt -days 100000 -extensions v3_ext -extfile ${SNAP_DATA}/certs/csr.conf
 }
+
+maybe_cache_juju_operator_images() {
+    if [ ! -f /snap/bin/juju.cache-images ]; then
```

Comment:
I remember I read somewhere that `/snap` is configurable, but I cannot find a link now. Is there an env variable we could use or could you please point me to some snapcraft doc saying `/snap/` is not configurable?

Reply:
oh, good point. I'll double check for an env var to use.

Reply:
FWIW classic snaps won't install unless `/snap/` exists, either as a directory or a symlink.

Reply:
This is safe then.